tests/integration/test_feedback.py

```diff
@@ -17,7 +18,7 @@
 def _setup():
     """Setups the test client."""
     global client
-    config.init_config("tests/config/valid_config.yaml")
+    config.reload_from_yaml_file("tests/config/valid_config.yaml")
```

Comment:
why did you change the scope in test_authorized.py, but not here?

Reply:
You mean scope -> `function`? Because this fixture loads defined config, but there are other fixtures `_with_disabled_feedback` and `_with_enabled_feedback` that controls the config in each test.

But it is probably better to have it defined as a `function` by default - these tests are not affected by the change.

Changed.